website/ref/language.md

```diff
@@ -2102,10 +2102,19 @@ This control structure behaves as follows:
     Example:
 
     ```elvish-transcript
-    ~> try { nop } else { echo well }
+    ~> try { nop } catch { echo bad } else { echo well }
     well
     ```
 
+    **Note**: `else` requires a `catch` or `finally` to be present. The
+    following code is invalid:
+
+    ```elvish-transcript
+    ~> try { nop } else { echo well }
+    Compilation error: try must be followed by a catch block or a finally block
+      [tty 1]:1:1-30: try { nop } else { echo well }
+    ```
+
```

Comment:
The text after the bullet list explains that either `catch` or `finally` must be present, so I'd move this explanation to be part of that.

Reply:
I made the change but it looks like this PR is already merged

I've created a new PR which addresses this comment, and also fixed some other issues with the command outputs I noticed

https://github.com/elves/elvish/pull/1783